binaries/scootapi/main.go

```diff
@@ -11,7 +11,7 @@ func main() {
 	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
 	transportFactory := thrift.NewTTransportFactory()
 
-	dialer := client.NewDialer("localhost:9090", transportFactory, protocolFactory)
+	dialer := client.NewDialer(transportFactory, protocolFactory)
```

Comment:
I think that would be better, however I'm willing to brainstorm about how to keep Dialer and not be shady.